src/builtin.c

```diff
@@ -226,6 +242,11 @@ static jv f_frexp(jq_state *jq, jv input) {
   jv_free(input);
   return ret;
 }
+#else
+static jv f_frexp(jq_state *jq, jv input) {
+  jv error = jv_string("Error: frexp/0 not found at build time");
+  return ret_error(input, error);
+}
```

Comment:
Worth adding some `LIBM_DD` macro thingy for functions that return array? maybe messy to make such macro generic?

Reply:
Sure, why not